helm_deploy/values-dev.yaml

```diff
@@ -65,3 +65,6 @@ generic-service:
    KEYWORKER_MAINTENANCE_MODE: false
    GET_SOMEONE_READY_FOR_WORK_URL: https://get-ready-for-work-dev.hmpps.service.justice.gov.uk
    WARRANT_FOLDER_URL: https://manage-a-warrant-folder-dev.hmpps.service.justice.gov.uk/
+   PRISONER_PROFILE_REDIRECT_URL: http://prisoner-dev.digital.prison.service.justice.gov.uk
+   PRISONER_PROFILE_REDIRECT_ENABLED: true
```

Comment:
Can we switch these to (in all environments):

```
   PRISONER_PROFILE_REDIRECT_ENABLED: false
   PRISONER_PROFILE_REDIRECT_ENABLED_PRISONS: LEI
```

We can test using the `DPS_APPLICATION_DEVELOPER` role before switching ENABLED: true